plugins/woocommerce/src/Blocks/BlockTypes/Reviews/ProductReviewForm.php

```diff
@@ -73,84 +75,142 @@ protected function render( $attributes, $content, $block ) {
-				 */
-				comment_form( apply_filters( 'woocommerce_product_review_comment_form_args', $comment_form ), $block->context['postId'] );
-				echo '</div></div>';
-				$form = ob_get_clean();
-
-				remove_filter( 'comment_form_defaults', 'post_comments_form_block_form_defaults' );
-
-				$form = str_replace( 'class="comment-respond"', $wrapper_attributes, $form );
-
-				wp_enqueue_script( 'comment-reply' );
-
-				return $form;
+		$fields              = [
+			'author' => [
+				'label'        => __( 'Name', 'woocommerce' ),
+				'type'         => 'text',
+				'value'        => $commenter['comment_author'],
+				'required'     => $name_email_required,
+				'autocomplete' => 'name',
+			],
+			'email'  => [
+				'label'        => __( 'Email', 'woocommerce' ),
+				'type'         => 'email',
+				'value'        => $commenter['comment_author_email'],
+				'required'     => $name_email_required,
+				'autocomplete' => 'email',
+			],
+		];
+
+		$comment_form['fields'] = [];
+
+		foreach ( $fields as $key => $field ) {
+			$field_html  = '<p class="comment-form-' . esc_attr( $key ) . '">';
+			$field_html .= '<label for="' . esc_attr( $key ) . '">' . esc_html( $field['label'] );
+
+			if ( $field['required'] ) {
+				$field_html .= '&nbsp;<span class="required">*</span>';
+			}
+
+			$field_html .= '</label><input id="' . esc_attr( $key ) . '" name="' . esc_attr( $key ) . '" type="' . esc_attr( $field['type'] ) . '" autocomplete="' . esc_attr( $field['autocomplete'] ) . '" value="' . esc_attr( $field['value'] ) . '" size="30" ' . ( $field['required'] ? 'required' : '' ) . ' /></p>';
+
+			$comment_form['fields'][ $key ] = $field_html;
+		}
+
+		$account_page_url = wc_get_page_permalink( 'myaccount' );
+		if ( $account_page_url ) {
+			/* translators: %s opening and closing link tags respectively */
+			$comment_form['must_log_in'] = '<p class="must-log-in">' . sprintf( esc_html__( 'You must be %1$slogged in%2$s to post a review.', 'woocommerce' ), '<a href="' . esc_url( $account_page_url ) . '">', '</a>' ) . '</p>';
+		}
+
+		if ( wc_review_ratings_enabled() ) {
+			$interactivy_state['selectedStar']            = '';
+			$interactivy_state['hoveredStar']             = '0';
+			$interactivy_state['ratingError']             = '';
+			$interactivy_state['hasRatingError']          = false;
+			$interactivy_config['i18nRequiredRatingText'] = esc_attr__( 'Please select a rating', 'woocommerce' );
+			$interactivy_config['reviewRatingRequired']   = wc_review_ratings_required();
+
+			$comment_form['comment_field'] = '<div class="comment-form-rating"><label for="rating-selector" id="comment-form-rating-label">' .
+				esc_html__( 'Your rating', 'woocommerce' ) . ( wc_review_ratings_required() ? '&nbsp;<span class="required">*</span>' : '' ) .
+				'</label><select name="rating" id="rating-selector" data-wp-init="callbacks.hideRatingSelector" data-wp-bind--value="state.selectedStar" ' . ( wc_review_ratings_required() ? ' required' : '' ) . '>
```

Comment:
That was an issue in the original template, so we might decide to leave it as it was. But I would add a space between the label and the select. Otherwise, they look very close to each other:

<img width="664" height="738" alt="imatge" src="https://github.com/user-attachments/assets/2efc2022-4fc4-43f0-aa8f-34f0322a5ffa" />

Reply:
Makes sense, I addressed in [4956e45](https://github.com/woocommerce/woocommerce/pull/60213/commits/4956e45cd79e9b3f6a690700d4996d7fd1e9e348)